src/main/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTask.java

```diff
@@ -152,6 +157,14 @@ public void setMaxWarnings(int maxWarnings) {
     }
 
     /**
+     * Adds a source path.
+     * @param sourcePath the source path to add.
+     */
+    public void addSourcePath(Path sourcePath) {
```

Comment:
I think that property should be just `path` as Checkstyle could validate not only java files (configs, properties, .....).